### PR TITLE
CL Briefcase Radio Keys

### PR DESCRIPTION
**Pull request by blackdragonTOW**
# About the pull request

CLs lost squad comms a while back and this has hampered some RP in select cases.  This re-adds them but at a cost.  Given that were pulling them from the briefcase, this gives the CL's case more value and more things to spend points on rather than the usual gimmicks.

Squad key costs = 25 each (if they want all 4 squads, they need 100 points)
Wey-Yu related groups = 30 each
Contraband groups = 50 each (note: this is 100% of the CLs max points) this is mostly to create RP as the CL can get caught with these keys.

Willing to have a conversation about key costs (CLs start with 50 (cap is 50) and generate 1 per minute, meaning for 100 points they need to dump all starting points and play for 50min.)

# Explain why it's good for the game

CL Briefcase's typically are meme spawns, this will allow for more interesting choices. 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: Added Radio Keys to the CL's briefcase
/:cl:

### CODE REVIEW

**Comment by blackdragonTOW** [comment]
After some brief conversations with people made the following changes:
-Across the board reduction by 10 points
-Color field wasn't working, so reverted to White
-Moved the Contraband warning tag from the info desc to the name of the item to make it obvious to the CL

File: code/game/objects/items/devices/portable_vendor.dm [hunk 1]
```diff
@@ -274,4 +274,16 @@
...
+		list("Bravo Squad", 15, /obj/item/device/encryptionkey/bravo, "white", "Radio Key for USCM Bravo Squad."),
+		list("Charlie Squad", 15, /obj/item/device/encryptionkey/charlie, "white", "Radio Key for USCM Charlie Squad."),	
+		list("Delta Squad", 15, /obj/item/device/encryptionkey/delta, "white", "Radio Key for USCM Delta Squad."),
+		list("Echo Squad", 15, /obj/item/device/encryptionkey/echo, "white", "Radio Key for USCM Echo Squad."),	
+		list("W-Y PMC", 20, /obj/item/device/encryptionkey/pmc, "white", "Radio Key for Weyland-Yutani PMC Combat Comms."),
+		list("Colony", 20, /obj/item/device/encryptionkey/colony, "white", "Pre-tuned Radio Key for local colony comms."),	
+		list("CONTRABAND: Colonial Marshals", 40, /obj/item/device/encryptionkey/cmb, "white", "Radio Key for the CMB."),	
+		list("CONTRABAND: Colonial Liberation Front", 40, /obj/item/device/encryptionkey/clf, "white", "Radio Key for known local CLF frequencies."),	
+		list("CONTRABAND: Union of Progressive Peoples", 40, /obj/item/device/encryptionkey/upp, "white", "Radio Key for known UPP listening frequencies."),
```
Comment (realforest2001) [inline code/game/objects/items/devices/portable_vendor.dm:288]:
```suggestion
		list("Colony", 20, /obj/item/device/encryptionkey/colony, "white", "Pre-tuned Radio Key for local colony comms."),	
```
Not having them able buy other faction comms as this will lead to problems I would rather we avoid. At most I would say put this on a toggle, so that it can be switched on in variables.

**Review by realforest2001 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/portable_vendor.dm (hunk 1 above)
Comment (blackdragonTOW) [reply code/game/objects/items/devices/portable_vendor.dm:288]: What sort of problems?

Comment (blackdragonTOW) [reply code/game/objects/items/devices/portable_vendor.dm]: The world may never know.   Moved contra to another section hidden behind a var that admins can change.

**Comment by Drulikar** [comment]
Mentioned this on discord, but to do what you were trying to do earlier with contraband, I'd recommend a separate list that your var can optionally display in addition the regular items. A way to do this would be like this: 
[patch.diff.txt](https://github.com/cmss13-devs/cmss13/files/13533519/patch.diff.txt)

File: code/game/objects/items/devices/portable_vendor.dm [hunk 2]
```diff
@@ -274,4 +298,20 @@
...
 		list("AMMO", 0, null, null, null),
 		list("ES-4 stun magazine", 10, /obj/item/ammo_magazine/pistol/es4, "white", "Holds 19 rounds of specialized Conductive 9mm."),
+
+		list("RADIO KEYS", 0, null, null, null),
+		list("Alpha Squad", 15, /obj/item/device/encryptionkey/alpha, "white", "Radio Key for USCM Alpha Squad."),	
+		list("Bravo Squad", 15, /obj/item/device/encryptionkey/bravo, "white", "Radio Key for USCM Bravo Squad."),
+		list("Charlie Squad", 15, /obj/item/device/encryptionkey/charlie, "white", "Radio Key for USCM Charlie Squad."),	
+		list("Delta Squad", 15, /obj/item/device/encryptionkey/delta, "white", "Radio Key for USCM Delta Squad."),
+		list("Echo Squad", 15, /obj/item/device/encryptionkey/echo, "white", "Radio Key for USCM Echo Squad."),	
+		list("Colony", 20, /obj/item/device/encryptionkey/colony, "white", "Pre-tuned Radio Key for local colony comms."),
```
Comment (Drulikar) [inline code/game/objects/items/devices/portable_vendor.dm:308]: Extra unnecessary whitespace (wouldn't have this problem if you didn't webedit).
```suggestion
		list("Alpha Squad", 15, /obj/item/device/encryptionkey/alpha, "white", "Radio Key for USCM Alpha Squad."),
		list("Bravo Squad", 15, /obj/item/device/encryptionkey/bravo, "white", "Radio Key for USCM Bravo Squad."),
		list("Charlie Squad", 15, /obj/item/device/encryptionkey/charlie, "white", "Radio Key for USCM Charlie Squad."),
		list("Delta Squad", 15, /obj/item/device/encryptionkey/delta, "white", "Radio Key for USCM Delta Squad."),
		list("Echo Squad", 15, /obj/item/device/encryptionkey/echo, "white", "Radio Key for USCM Echo Squad."),
		list("Colony", 20, /obj/item/device/encryptionkey/colony, "white", "Pre-tuned Radio Key for local colony comms."),
```

**Review by Drulikar — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/portable_vendor.dm (hunk 2 above)
Comment (blackdragonTOW) [reply code/game/objects/items/devices/portable_vendor.dm:308]: wild

**Review by Drulikar — APPROVED** [review]
